### PR TITLE
image pull secret

### PR DESCRIPTION
**Pull request by cp4s-cases**
configurable image pull secret

### CODE REVIEW

File: ansible/inventory [hunk 1]
```diff
@@ -106,24 +109,20 @@ db_user='testuser'
 # Which storage definitions to use when creating persistent volumes
 # for a variety of uses.
 
-backrest_storage='storage1'
-backup_storage='storage1'
-primary_storage='storage2'
+backrest_storage='storage2'
+backup_storage='storage2'
```
Comment (JohnMalmsteen) [inline ansible/inventory:113]: is there something that actually performs a backup or is it a manual pgo client interaction?

File: ansible/inventory [hunk 2]
```diff
@@ -106,24 +109,20 @@ db_user='testuser'
...
+storage1_class='glusterfs-storage'
 
 storage2_access_mode='ReadWriteOnce'
-storage2_size='1G'
+storage2_size='200G'
 storage2_type='dynamic'
-storage2_class='fast'
-#storage2_supplemental_groups=65534
-#storage2_fs_group=26
+storage2_class='glusterfs-storage'
```
Comment (JohnMalmsteen) [inline ansible/inventory:125]: is this documented somewhere that a user may have to change it?

File: ansible/inventory (hunk 1 above)
Comment (cp4s-cases) [reply ansible/inventory:113]: we wont have backup enabled initially. We can state this explicitly in documentation.

File: ansible/inventory (hunk 2 above)
Comment (cp4s-cases) [reply ansible/inventory:125]: we reference main Crunchy documentation

**Review by JohnMalmsteen — APPROVED** [review]

**Pull request merged**